### DISCUSSION TITLE
Installing WarpX on a Computing Center's Server: Conda or Source Code Compilation?

### DISCUSSION THREAD

**Discussion by jinze-liu** [BLAST-WarpX/warpx#4822] [Installation] [answered]
I successfully installed and ran WarpX on my personal computer's Linux system using Conda. Now, I want to install and run WarpX on a computing center's server. Can I use Conda to install WarpX on the computing center's server, or do I need to compile it from the source code?

**Comment by ax3l** [comment, marked as answer]
Hi @jinze-liu,

Thanks for the question! For HPC systems, we recommend to compile WarpX from source. This is, because you might want to run on multiple nodes using the center's compilers, modules, MPI (tuned for their hardware), their GPUs, etc.

We have many examples for HPC systems here:
https://warpx.readthedocs.io/en/latest/install/hpc.html

and if you like to share and document your HPC center, please do not hesitate to open a pull request adding it and we will help you to get to a working solution.
